proxy/ParentConsistentHash.cc

```diff
@@ -61,22 +61,22 @@ ParentConsistentHash::~ParentConsistentHash()
 uint64_t
 ParentConsistentHash::getPathHash(HttpRequestData *hrdata, ATSHash64 *h)
 {
-  const char *tmp = NULL;
+  const char *url_string_ref = NULL;
   int len;
   URL *url = hrdata->hdr->url_get();
 
   // Use over-ride URL from HttpTransact::State's cache_info.parent_selection_url, if present.
   URL *ps_url = NULL;
-  Debug("parent_select", "hrdata->cache_info_parent_selection_url = %p", hrdata->cache_info_parent_selection_url);
+  // Debug("parent_select", "hrdata->cache_info_parent_selection_url = %p", hrdata->cache_info_parent_selection_url);
```

Comment:
We shouldn't be commenting out code. We have the history in git. This line should be totally removed or stay uncommented.